.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Release Pre-Check
+on:
+  pull_request:
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  release-suite:
+    runs-on: ubuntu-latest
+    name: Run googleapis/google-cloud-php tests against latest version
+    # @TODO: Uncomment before merge
+    # if: github.event.pull_request.user.login == 'release-please[bot]'
+    steps:
+    - uses: actions/checkout@v4
+    - name: Clone googleapis/google-cloud-php
+      uses: actions/checkout@master
+      with:
+        repository: googleapis/google-cloud-php
+        path: google-cloud-php
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '8.1'
+        extensions: grpc
+    - name: Configure GAX dev-main
+      # @TODO change version to be "dev-main" before we merge
+      run: |
+        cd google-cloud-php
+        composer install -q -d dev
+        dev/google-cloud update-deps google/gax 'dev-run-google-cloud-php-tests as 1.200.0' --add=dev
```

Depends on https://github.com/googleapis/gax-php/pull/503
```suggestion
        dev/google-cloud update-deps google/gax 'dev-main as 1.200.0' --add=dev
```